aws/rust-runtime/aws-runtime/src/auth.rs

```diff
@@ -168,11 +164,11 @@ pub mod sigv4 {
             settings: SigningSettings,
             credentials: &'a Credentials,
             operation_config: &'a SigV4OperationSigningConfig,
+            request_timestamp: SystemTime,
         ) -> SigningParams<'a> {
             if let Some(expires_in) = settings.expires_in {
                 if let Some(creds_expires_time) = credentials.expiry() {
-                    let presigned_expires_time =
-                        operation_config.signing_options.request_timestamp + expires_in;
+                    let presigned_expires_time = request_timestamp + expires_in;
                     if presigned_expires_time > creds_expires_time {
```

Comment:
Possible alternate word: [expiry](https://www.merriam-webster.com/dictionary/expiry)
```suggestion
                    let presigned_expiry = request_timestamp + expires_in;
                    if presigned_expiry > creds_expiry {
```